circle.yml

```diff
@@ -69,3 +69,15 @@ deployment:
           (test "${DOCKER_ORGANIZATION:-$DOCKER_USER}" == "weaveworks" || docker tag weaveworks/scope:latest ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:latest) &&
           docker push ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope
         )
+  release:
+    branch: release-/[0-9]+\.[0-9]+/
```

Comment:
Do you know for certain this will work? Regexp matches in circle have to be bracketed with `/../` I would have thought it would be `/release-[0-9]+\.[0-9]+/`